modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMHomePage.testcase

```diff
@@ -49,4 +49,41 @@ definition {
 		}
 	}
 
+	@description = "LPS-172079 - Verify that the admin user can add a new announcement."
+    @priority = "5"
+    test AdminCanAddAnnouncement {
+
+        task ("When: The User goes to announcements in web content"){
+
+        WebContentNavigator.openWebContentAdmin(siteURLKey = "partner-portal");
+
+        PRMUtils.gotoFolder(folderName = "Announcements");
+        }
+
+        task("Then: User add an announcement and publish it"){
+        LexiconEntry.gotoAddMenuItem(menuItem = "Announcement");
+
+        PortletEntry.inputTitle(title = "Announcement Title");
+
+        CKEditor.addContentInWebContent(
+            content = "Announcement Body",
+            fieldLabel = "Body");
+
+        Button.click(button = "Publish");
+        
+        Alert.viewSuccessMessage();
+        }
+        task("Then: The user then goes back to the home page and check if the ad was published"){
```

Comment:
Hey @marinaasant0s, I think that we can improve this task.
It could be: "Then: The announcement is visible on PRM Home Page"